bazel/external/quiche.BUILD

```diff
@@ -216,6 +224,30 @@ cc_library(
     ],
 )
 
+envoy_cc_test(
+    name = "http2_platform_test",
+    srcs = envoy_select_quiche(
+        [
```

Comment:
Please merge make the use of brackets match other places in Envoy. Prefer compact style as with google3 C++ and BUILD files.

Reply:
Done.